src/roles/common/templates/hosts.j2

```diff
@@ -258,6 +258,13 @@ vstat_node{{ loop.index }}
 {% endfor %}
 {% endif %}
 
+{% if netconf_managers is defined and netconf_managers|length > 0 %}
+[netconf_managers]
+{% for netconf_manager in netconf_managers %}
+netconf_{{ netconf_manager.netconf_server }}
```

Comment:
I don't think we should introduce the prefix on the host name. It's okay for a host to appear in more than one group, so dual-use VMs will still be supported.

Reply:
We had to do a special case for this for one reason. The VM name they specify could be same as one of the other VMs for the NUH or VNSUtil and we will override the host vars property of that VM. So this has to be done.